modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFClaim.macro

```diff
@@ -68,6 +68,23 @@ definition {
 		WaitForPageLoad();
 	}
 
+	@summary = "Get the claim ID by index"
+	macro filterClaimID {
```

Comment:
Hi @marinaasant0s, this macro summary and name doesn't match. Could you please double-check it?

Reply:
yes, sorry, I corrected that.